src/test/resources/org/rust/ide/intentions/fixtures/add_derive/add_derive_enum_after.rs

```diff
@@ -0,0 +1,3 @@
+    #[derive(<caret>)]enum Test {
+        Something
+    }
```

Comment:
There is one more thing in this test: a newline should be required between `#[dervive ]` and `enum`. This can be specified in `Spacing`.